### PR TITLE
Widen grpc range, fix windows

### PR DESCRIPTION
**Pull request by h-vetinari**
Follow-up to #70 / https://github.com/ray-project/ray/issues/28354, and the windows build failures seen in #73

### CODE REVIEW

**Review by krfricke — APPROVED** [review]

**Comment by h-vetinari** [comment]
Something weird happening with bazel vs. JAVA_HOME.

**Comment by mattip** [comment]
The error is
```
Extracting Bazel installation...
WARNING: Ignoring JAVA_HOME, because it must point to a JDK, not a JRE.
FATAL: Could not find system javabase. Ensure JAVA_HOME is set, or javac is on your PATH.
```

Let's try this again.

**Comment by mattip** [comment]
@conda-forge-admin, please rerender

**Comment by mattip** [comment]
JAVA_HOME is set to `<build_env>\Library\lib\jvm`. It should be set to `<buildenv>\Library\bin`, where `javac.exe` lives.

**Comment by mattip** [comment]
So it seems the problem is that openjdk=11 on windows does not package the JRE files (all the *.jar files that appear in openjdk on linux under JAVA_HOME). However openjdk=8 does properly package them, and bazel=4.2.2 functions. (I can do `bazel --help`)

File: recipe/meta.yaml [hunk 1]
```diff
@@ -72,6 +72,7 @@ outputs:
         # pickle5 needs to be here, otherwise setup.py tries
         # to install it and it fails on conda-forge Windows CI
         - pickle5  # [py<38]
+        - openjdk =11
```
Comment (mattip) [inline recipe/meta.yaml:75]:
```suggestion
        - openjdk =11 [not win]
        - openjdk =8 [win]
```

**Comment by h-vetinari** [comment]
Problem was https://github.com/conda-forge/openjdk-feedstock/issues/107, those builds have been pulled. Let's see if it works now.

**Comment by mattip** [comment]
Cool, thanks. It seems the windows builds got past the line that failed last time. Keeping my fingers crossed..

**Pull request merged**